azurerm/internal/services/applicationinsights/application_insights_analytics_item_resource.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 	"time"
 
-	"github.com/Azure/azure-sdk-for-go/services/appinsights/mgmt/2015-05-01/insights"
+	"github.com/Azure/azure-sdk-for-go/services/appinsights/mgmt/2020-02-02/insights"
```

Comment:
normal practice is to update api versions in separate PR before doing functional changes

these sometimes come with regressions and it can be better to isolate changes

Reply:
good shoutout and typically yes, but its a judgment call i always make on the size & scope of the service and change. This is a small PR and service with 100% passing tests so not to concerned by it!

Reply:
Thanks @timja @katbyte - will keep in mind for future PRs.